notifications/templates/report.html

```diff
@@ -0,0 +1,144 @@
+</div>
+<p>
+    Hier ist deine Getränkeübersicht seit dem {{ since_text }}.
+    <br>
+    Dein aktuelles Guthaben beträgt
+    {% if balance > 0 %}
+    +{{ balance }}€.
+    {% else %}
+    {{ balance }}€.
+    {% endif %}
+</p>
+<h2>
+    Abbuchungen
+</h2>
+<table>
+    <thead>
+        <tr>
+            <th>
+                #
+            </th>
+            <th>
+                Datum
+            </th>
+            <th>
+                Getränk
+            </th>
+            <th>
+                Größe
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        {% for drink in drinks %}
+        <tr>
+            <td>
+                {{ loop.index }}
+            </td>
+            <td>
+                {{ drink.timestamp.strftime("%F %T Z") }}
+            </td>
+            <td>
+                {{ drink.name }}
+            </td>
+            <td>
+                {{ drink.size }} l
+            </td>
+        </tr>
+        {% endfor %}
+    </tbody>
+</table>
+<h2>
+    Aufladungen
+</h2>
+<table>
+    <thead>
+        <tr>
+            <th>
+                #
+            </th>
+            <th>
+                Datum
+            </th>
+            <th>
+                4-Augen-Person
+            </th>
+            <th>
+                Wert
+            </th>
+        </tr>
+    </thead>
+    <tbody>
+        {% for recharge in recharges %}
+        <tr>
+            <td>
+                {{ loop.index }}
+            </td>
+            <td>
+                {{ recharge.timestamp.strftime("%F %T Z") }}
+            </td>
+            <td>
+                {{ recharge.helper_user_id }}
+            </td>
+            <td>
+                {{ recharge.amount }}€
+            </td>
+        </tr>
+        {% endfor %}
+    </tbody>
+</table>
+<hr>
+<p class="text-centered">
+    Besuchen Sie uns bald wieder!
+</p>
+<div id="buttons">
+    <form action="http://ldap.fd/">
+        <button title="http://ldap.fd/" type="submit">
+            <span>
+                Einstellungen
+                <img alt="open in new" class="open-in-new" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABIAAAASCAQAAAD8x0bcAAAAe0lEQVR4Aa3RAQaAQBBA0b8SoFvUvfY2AQqEEIEgnWCxRwoQYaIYhrGgP7DsYzD8XEci60DU94i2czroRsho35dtRrhKKLykJztIyYowgI8sgUj0UGBDmDBZVFniras4EBYooZrMQigjaAxxkOYjcxYzJztaS3JRouXXHoGiUs6HmNWDAAAAAElFTkSuQmCC">
+            </span>
+        </button>
+    </form>
+    <form action="http://drinks-touch.fd/">
+        <button title="http://drinks-touch.fd/" type="submit">
+            <span>
+                Aufladen
+                <img alt="open in new" class="open-in-new" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABIAAAASCAQAAAD8x0bcAAAAe0lEQVR4Aa3RAQaAQBBA0b8SoFvUvfY2AQqEEIEgnWCxRwoQYaIYhrGgP7DsYzD8XEci60DU94i2czroRsho35dtRrhKKLykJztIyYowgI8sgUj0UGBDmDBZVFniras4EBYooZrMQigjaAxxkOYjcxYzJztaS3JRouXXHoGiUs6HmNWDAAAAAElFTkSuQmCC">
+            </span>
+        </button>
+    </form>
+</div>
+<table id="sepa">
```

Comment:
please make sure to include the sepa info in every mail (main.html?) and use a jinja include instead of duplicating code